src/distributions/uniform.rs

```diff
@@ -157,40 +158,40 @@ pub trait RangeImpl: Sized {
     /// and exclusive upper bound `[low, high)`.
     ///
     /// Usually users should not call this directly but instead use
-    /// `Range::sample_single`, which asserts that `low < high` before calling
+    /// `Uniform::sample_single`, which asserts that `low < high` before calling
     /// this.
     ///
-    /// Via this method range implementations can provide a method optimized for
-    /// sampling only a limited number of values from range. The default
-    /// implementation just sets up a range with `RangeImpl::new` and samples
+    /// Via this method, implementations can provide a method optimized for
+    /// sampling only a limited number of values from the range. The default
```

Comment:
I guess this should say "for sampling only a single value from the specified range", but not actually a problem with your PR.